src/engraving/libmscore/tie.cpp

```diff
@@ -501,7 +650,6 @@ void Tie::slurPos(SlurPos* sp)
     qreal x1, y1;
     qreal x2, y2;
     bool shortStart = false;
```

Comment:
As far as I can tell this variable is not used at all, except for being set here and a fe lines further down.